libs/pages/application/src/lib/feature/page-settings-deployment-restrictions-feature/page-settings-deployment-restrictions-feature.tsx

```diff
@@ -121,69 +118,57 @@ function PageSettingsDeploymentRestrictionsFeatureInner({
   }
 
   return (
-    <>
+    <div>
```

Comment:
Why adding this `<div>` ?

Reply:
To avoid this kind of warning, more simple during my refactoring, I can revert 

<img width="898" alt="Screenshot 2024-04-30 at 15 45 37" src="https://github.com/Qovery/console/assets/533928/1cae405c-af86-4825-9bca-7fd2335af050">

Reply:
I've simplified it a bit
https://github.com/Qovery/console/pull/1322/commits/49b433d374c4647ddf15cd5d4a20cb45239cd3d1